lib/spack/spack/cmd/debug.py

```diff
@@ -64,7 +65,7 @@ def create_db_tarball(args):
     transform_args = []
     if "GNU" in tar("--version", output=str):
         transform_args = ["--transform", "s/^%s/%s/" % (base, tarball_name)]
-    else:
+    elif sys.platform != "win32":
```

Comment:
This looks good. Can we just leave a comment that indicates that neither `--transform` or `-s` are supported by Windows native tar at this point in time?

Reply:
Sure!